docs/develop/serpentVersions.rst

```diff
@@ -125,3 +125,31 @@ layout of output files, such that present readers will not work, then
 version-specific read methods will have to be written.
 If/when this happens, a procedure will be developed and added here.
 
+MapStrVersions
+----------------
+
+This is a mapping variable located in the ResultsReader parser. 
+The variable is a nested dictionary, where the keys describe the version and the nested
+dictionary describes the various data blocks within the results file (_res.m). 
+
+In general the results file is divided into three main blocks:
+- metadata: general description of the simulation
+- resdata: time-dependent values, e.g. k-eff
+- universes: universe dependent values, such as cross-sections
+The mapping through the variable `MapStrVersions` should reflect this. 
+
+The basic mapping definition relies on the following structure:
```

Comment:
Add a second colon here and tab the whole block in, like
```
.....structure::

    MapStrVersions = { ....

```

Further reading - http://www.sphinx-doc.org/en/stable/rest.html?highlight=url%20#source-code

This way the code will be rendered properly in the documentation. The empty line that separates the first line of code from the first line of text is crucial as well